docs/usage.rst

```diff
@@ -69,6 +69,42 @@ type of repo that you want to use prepending `hg+` or `git+` to repo url::
 
     $ cookiecutter hg+https://example.com/repo
 
+Works with Zip files
```

Comment:
I love that you always write great documentation for your changes, @freakboy3742!!! 📝 🙇